policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/cost_of_living/base.yaml

```diff
@@ -0,0 +1,9 @@
+description: Quebec cost of living tax credit base amount.
+values:
+  2022-01-01: 600
+metadata:
+  unit: currency-CAD
+  label: Quebec cost of living tax credit base amount 
+  reference:
```

Comment:
Add a legislative reference if possible, or if not please leave a comment saying it wasn't available.